### PR TITLE
Add article about intersectionobserver

### PR DESCRIPTION
**Pull request by surma**
cc @PaulKinlan @jeffposnick @ojanvafai

### CODE REVIEW

File: src/content/en/updates/posts/2016/04/intersectionobserver.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+layout: updates/post
+title: "IntersectionObserver"
```
Comment (jeffposnick) [inline src/content/en/updates/posts/2016/04/intersectionobserver.md]: I'd go with a catchier title.

"Watch Out for IntersectionObserver" or "IntersectionObserver's Coming into View" or something along those lines.

File: src/content/en/updates/posts/2016/04/intersectionobserver.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+description: "IntersectionObservers let you know when an observed element enters or exits the browser’s viewport."
+published_on: 2016-04-21
+updated_on: 2016-04-21
+authors:
+  - surma
+tags:
+  - intersectionobserver
+---
+
+# Intersection Observer Web Update
```
Comment (jeffposnick) [inline src/content/en/updates/posts/2016/04/intersectionobserver.md]: Intersection Observer → IntersectionObserver

File: src/content/en/updates/posts/2016/04/intersectionobserver.md (hunk 1 above)
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/04/intersectionobserver.md]: "IntersectionObserver's Coming into View" YES

File: src/content/en/updates/posts/2016/04/intersectionobserver.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+{% endhighlight %}
+
+# iframe magic
+`IntersectionObservers` were designed specifically with ads services and social network widgets in
+mind, which frequently use iframes and could benefit from knowing whether they
+are in view. If an iframe observes one of its elements, both scrolling the
+iframe as well as scrolling the window *containing the iframe* will trigger the callback at the appropriate times. For the latter case,
+however, `rootBounds` will be set to `null` to avoid leaking data across origins.
+
+# What is `IntersectionObserver` *not* about?
```
Comment (jeffposnick) [inline src/content/en/updates/posts/2016/04/intersectionobserver.md]: Remove the backticks:
image

File: src/content/en/updates/posts/2016/04/intersectionobserver.md (hunk 1 above)
Comment (surma) [inline src/content/en/updates/posts/2016/04/intersectionobserver.md]: Got no one but yourself to blame

**Comment by surma** [comment]
Done. Thanks @jeffposnick! PTAL

File: src/content/en/updates/posts/2016/04/intersectionobserver.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
+---
+layout: updates/post
+title: "IntersectionObserver’s coming into view"
+description: "IntersectionObservers let you know when an observed element enters or exits the browser’s viewport."
+published_on: 2016-04-21
+updated_on: 2016-04-21
+authors:
+  - surma
+tags:
+  - intersectionobserver
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/04/intersectionobserver.md]: - chrome51

**Comment by jeffposnick** [comment]
LGTM once you add in the `chrome51` tag. Cheers.

**Comment by surma** [comment]
Done :)

**Pull request merged**